index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/styles.css" >
+    <title>Portifolio</title>
+</head>
+<body>
+    <header>
+        <div class="top-head">
+            <p class="logo">
+                <a href="" class="logotext">Welcome</a>
+            </p>
+            <nav>
+                <div class="hamburger">
+                    <img src="./images/menuicon.png" alt="menuicon" class="icon">
+                </div>  
+            </nav>
+        </div>
+        
+    </header>
+    <section class="contents">
+        <div class="headline">
+            <div class="primary">
+                <div class="primaryblack">
+                    <h4>Hey, I'm Aidan</h4>
```

Comment:
- [ ] Kindly change the `h4` tag to `h1` tag because it's always good to add one `h1` tag on a web page for good readability.

Reply:
Resolved